### PR TITLE
Standardize See also sections

### PR DESCRIPTION
**Pull request by nxtn**
Contributes to #6923 

Fixes 3 commented out links.

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/windows-workflow-foundation/bufferreceive.md [hunk 1]
```diff
@@ -44,5 +44,5 @@ A service behavior that enables a service to use buffered receive processing, wh
 |[\<behavior> of \<serviceBehaviors>](../../../../../docs/framework/configure-apps/file-schema/windows-workflow-foundation/behavior-of-servicebehaviors-of-workflow.md)|Specifies a behavior element.|  
   
 ## See also
-<!-- <xref:System.ServiceModel.Activities.Description.BufferReceiveServiceBehavior>  -->
+- <xref:System.ServiceModel.Activities.Description.BufferedReceiveServiceBehavior>
```
Comment (NextTurn) [inline docs/framework/configure-apps/file-schema/windows-workflow-foundation/bufferreceive.md]: Another "fix" for build warnings.

File: docs/core/porting/project-structure.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ Changes to note are:
 
 ## See also
 
-* Please see the [.NET Core porting documentation][porting-doc] for more guidance on migrating to .NET Core.
+Please see the [.NET Core porting documentation][porting-doc] for more guidance on migrating to .NET Core.
 
 [porting-doc]: index.md
```
Comment (NextTurn) [inline docs/core/porting/project-structure.md]: These links don't work at all.

File: docs/standard/cross-platform/windowsruntimestreamextensions-asrandomaccessstream-method.md [hunk 3]
```diff
@@ -70,5 +70,5 @@ Supported in: Windows 8.1
 
 ## See also
 
--[System.IO.WindowsRuntimeStreamExtensions](https://msdn.microsoft.com/library/system.io.windowsruntimestreamextensions(v=vs.110).aspx)
--[How to: Convert Between .NET Framework Streams and Windows Runtime Streams](../io/how-to-convert-between-dotnet-streams-and-winrt-streams.md)
+- [System.IO.WindowsRuntimeStreamExtensions](https://msdn.microsoft.com/library/system.io.windowsruntimestreamextensions(v=vs.110).aspx)
```
Comment (rpetrusha) [inline docs/standard/cross-platform/windowsruntimestreamextensions-asrandomaccessstream-method.md]: This item can be removed; WindowsRuntimeStreamExtensions isn't listed in the System.IO namespace.

File: docs/visual-basic/getting-started/index.md [hunk 4]
```diff
@@ -27,32 +27,33 @@ Lists changes in this release that might prevent an application from compiling o
 Provides a list of Web sites and newsgroups that can help you find answers to common problems.  
 
 ## See also
+
 - [Get Visual Basic](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs)
- Provides download links for Visual Studio versions that include Visual Basic support, including free versions.  
+ Provides download links for Visual Studio versions that include Visual Basic support, including free versions.
```
Comment (rpetrusha) [inline docs/visual-basic/getting-started/index.md]: It would be best for the description to be list continuation text on a separate line from the link. Since the link and the description are on the same line, they're hard to parse:
image

File: docs/visual-basic/misc/root-folder-cannot-be-renamed.md [hunk 5]
```diff
@@ -11,8 +11,8 @@ An attempt has been made to rename the root directory. This may be a result of a
 -   Check the file path to make sure that it is correct.  
   
 ## See also
- [My.Computer.FileSystem.RenameDirectory]xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.RenameDirectory)  
- [My.Computer.FileSystem.RenameFile]xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.RenameFile)  
+- [My.Computer.FileSystem.RenameDirectory](xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.RenameDirectory)
```
Comment (rpetrusha) [inline docs/visual-basic/misc/root-folder-cannot-be-renamed.md]: The link is broken. It should be xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.RenameDirectory%2A

File: docs/visual-basic/misc/root-folder-cannot-be-renamed.md [hunk 6]
```diff
@@ -11,8 +11,8 @@ An attempt has been made to rename the root directory. This may be a result of a
...
+- [My.Computer.FileSystem.RenameFile](xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.RenameFile)
```
Comment (rpetrusha) [inline docs/visual-basic/misc/root-folder-cannot-be-renamed.md]: The link is broken; it should be xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.RenameFile%2A

File: docs/visual-basic/misc/the-folder-cannot-be-created-since-a-file-already-exists-with-the-same-path.md [hunk 7]
```diff
@@ -15,6 +15,6 @@ An attempt has been made to create a directory, but a file already exists with t
 ## See also
 - [How to: Create a Directory](../../visual-basic/developing-apps/programming/drives-directories-files/how-to-create-a-directory.md)
 - [My.Computer.FileSystem](xref:Microsoft.VisualBasic.FileIO.FileSystem)
- [My.Computer.FileSystem.CreateDirectory]xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.CreateDirectory)  
- [My.Computer.FileSystem.RenameDirectory]xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.RenameDirectory)  
+- [My.Computer.FileSystem.CreateDirectory](xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.CreateDirectory)
```
Comment (rpetrusha) [inline docs/visual-basic/misc/the-folder-cannot-be-created-since-a-file-already-exists-with-the-same-path.md]: The xref should be xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.CreateDirectory

File: docs/visual-basic/misc/the-folder-cannot-be-created-since-a-file-already-exists-with-the-same-path.md [hunk 8]
```diff
@@ -15,6 +15,6 @@ An attempt has been made to create a directory, but a file already exists with t
...
+- [My.Computer.FileSystem.RenameDirectory](xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.RenameDirectory)
```
Comment (rpetrusha) [inline docs/visual-basic/misc/the-folder-cannot-be-created-since-a-file-already-exists-with-the-same-path.md]: The xref should be xref:Microsoft.VisualBasic.MyServices.FileSystemProxy.RenameDirectory%2A

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thank you, @NextTurn, for standardizing the See also section in these articles, as well as for fixing the migration issue with cross-references to the Strong Name tool. This looks really good, although I've left a number of comments for you to consider.

File: docs/core/porting/project-structure.md (hunk 2 above)
Comment (NextTurn) [reply docs/core/porting/project-structure.md]: Why not merge the links from line 68 to line 80 into the article?

Comment (NextTurn) [reply docs/core/porting/project-structure.md]: Why not inline the links from line 68 to line 80?

Comment (mairaw) [reply docs/core/porting/project-structure.md]: Yeah, you can do that. I actually prefer inline. I think the PM who wrote this liked this style of link.

File: docs/visual-basic/getting-started/index.md (hunk 4 above)
Comment (NextTurn) [reply docs/visual-basic/getting-started/index.md]: Should I use double spaces or `<br />` to break the line?

Comment (mairaw) [reply docs/visual-basic/getting-started/index.md]: What I've started doing in these cases where is to have a \ at the end of the line like

```markdown
- [Get Visual Basic](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs)\
Provides download links for Visual Studio versions that include Visual Basic support, including free versions.
```

Comment (NextTurn) [reply docs/visual-basic/getting-started/index.md]: I never knew this syntax. What does it mean?

Comment (mairaw) [reply docs/visual-basic/getting-started/index.md]: It's somewhat new to me as well but it breaks the line as you can see how the same content renders:

- [Get Visual Basic](https://aka.ms/vsdownload?utm_source=mscom&utm_campaign=msdocs)\
Provides download links for Visual Studio versions that include Visual Basic support, including free versions.

Comment (Thraka) [reply docs/visual-basic/getting-started/index.md]: Awesome suggestion @mairaw I didn't know about that either. Must nicer than the double spaces.

Comment (mairaw) [reply docs/visual-basic/getting-started/index.md]: I learned that with @mvelosop 😉

Comment (NextTurn) [reply docs/visual-basic/getting-started/index.md]: Double line breaks are used in the corresponding [C# article](https://github.com/dotnet/docs/blob/master/docs/csharp/getting-started/index.md).

**Comment by NextTurn** [comment]
I've addressed the changes @rpetrusha. Is it OK?

**Review by rpetrusha — APPROVED** [review]

**Comment by NextTurn** [comment]
@mairaw if you are to remove the "changes-addressed" labels on closed PRs, there are 10 of them.

https://github.com/dotnet/docs/issues?q=label%3Achanges-addressed+is%3Aclosed

**Comment by mairaw** [comment]
Done @NextTurn!

**Comment by NextTurn** [comment]
And 54 "verify-build-before-merge"

https://github.com/dotnet/docs/pulls?q=is%3Aclosed+label%3Averify-build-before-merge

**Comment by Thraka** [comment]
Done @NextTurn
